spark-partition-sizing/src/test/scala/za/co/absa/spark/partition/sizing/sizer/FromDataframeSampleSizerTest.scala

```diff
@@ -17,7 +17,7 @@
 package za.co.absa.spark.partition.sizing.sizer
 
 import org.scalatest.funsuite.AnyFunSuite
-import za.co.absa.spark.partition.sizing.DummyDatasets
+import za.co.absa.spark.partition.sizing.{DummyDatasets, ResourceData}
```

Comment:
`DummyDatasets` apparently already includes the `ResourceData` trait (should it? if really so, then the import is unnecessary)

There are multiple occurrences of this.